src/wallet/api/wallet.cpp

```diff
@@ -599,7 +599,7 @@ bool WalletImpl::open(const std::string &path, const std::string &password)
             // Rebuilding wallet cache, using refresh height from .keys file
             m_rebuildWalletCache = true;
         }
-        m_wallet->set_ring_database(get_default_ringdb_path());
+        m_wallet->get_ring_database();
```

Comment:
That does nothing. Is this intended ?

Reply:
I may be mistaken as I'm not that experienced with C but from my understanding before it was just setting the ring db path to the same default path in that function, but I refactored it to do the same logic unless the path is already set, allowing someone to manually change the path using the other setter, and now it won't overwrite it back to the default. Feel free to tell me I'm wrong haha